src/native/http_request_response.c

```diff
@@ -501,34 +425,26 @@ static int s_native_outgoing_body_read(struct aws_input_stream *input_stream, st
 
     JNIEnv *env = aws_jni_get_thread_env(callback->jvm);
 
-    uint8_t *out = &(dst->buffer[dst->len]);
     size_t out_remaining = dst->capacity - dst->len;
 
-    size_t buf_capacity = callback->native_body_buf.capacity;
-    size_t request_size = (buf_capacity > out_remaining) ? out_remaining : buf_capacity;
-
-    jobject jByteBuffer = callback->java_body_buf;
-
-    aws_jni_byte_buffer_set_position(env, jByteBuffer, 0);
-    aws_jni_byte_buffer_set_limit(env, jByteBuffer, (jint)request_size);
```

Comment:
These two functions can be deleted from crt.c as well, no one needs them anymore.

Reply:
These two appear to be used, I did see some others we could delete though.